test/unit/foreman/renderer/snapshots/ProvisioningTemplate/finish/Preseed_default_finish.debian_host4dhcp.snap.txt

```diff
@@ -60,14 +54,17 @@ environment     = test
 EOF
 
 
-puppet_unit=puppet
-/usr/bin/systemctl list-unit-files | grep -q puppetagent && puppet_unit=puppetagent
-/usr/bin/systemctl enable ${puppet_unit}
+if [ -f "/etc/default/puppet" ]
+then
+/bin/sed -i 's/^START=no/START=yes/' /etc/default/puppet
+fi
+/usr/bin/puppet agent --enable
 
 # export a custom fact called 'is_installer' to allow detection of the installer environment in Puppet modules
 export FACTER_is_installer=true
 # passing a non-existent tag like "no_such_tag" to the puppet agent only initializes the node
 /usr/bin/puppet agent --config /etc/puppet/puppet.conf --onetime --tags no_such_tag --server localhost --no-daemonize
+systemctl enable puppet
 
 
 
```

Comment:
To explain, preseed and PXELinux templates were in the list of snapshots, but when there is no fixture to use, the snapshot service uses "host4dhcp" by default which is EL7 host. Therefore the contents did not make any sense (see that yum was used instead of apt-get). This fixes it.